src/boot_O2_g3/fault.c

```diff
@@ -376,19 +462,20 @@ void Fault_PrintThreadContext(OSThread* t) {
     FaultDrawer_Printf("\n");
     FaultDrawer_SetCharPad(0, 0);
 
-    if (D_8009BE54 != 0) {
-        FaultDrawer_DrawText(0xA0, 0xD8, "%5.2f sec\n", D_8009BE54);
+    if (D_8009BE54 != 0.0f) {
+        FaultDrawer_DrawText(160, 216, "%5.2f sec\n", D_8009BE54);
```

Comment:
Hmm, I wonder if we can name `D_8009BE54` based on this. It seems like a timer/time taken value (think OoT speed_meter uses `TimeTotal`) of some kind, but doesn't seem to get set anywhere so always seems to be 0?

Reply:
It isn't set anywhere so it is always zero.
Maybe we could name it something like `sFaultTimer`?

Reply:
tbh, I'd probably go with `TimeTotal` just because it would seem more like it keeps track of how long something takes, though am fine with `timer`. I do think it should have an unused comment as well.

Reply:
Annoyingly not even the debug rom sets this variable